Source/Tools/FEXServer/ProcessPipe.cpp

```diff
@@ -189,7 +189,7 @@ namespace ProcessPipe {
     // Create the initial unix socket
     ServerSocketFD = socket(AF_UNIX, SOCK_STREAM | SOCK_CLOEXEC, 0);
     if (ServerSocketFD == -1) {
-      LogMan::Msg::AFmt("Couldn't create AF_UNIX socket: %d %s\n", errno, strerror(errno));
+      LogMan::Msg::EFmt("Couldn't create AF_UNIX socket: %d %s\n", errno, strerror(errno));
```

Comment:
Seems like this and the format message below is using printf specifiers instead of fmt specifiers

Reply:
oops, these weren't ever converted.

Reply:
Resolved.